include/lldb/Symbol/SwiftASTContext.h

```diff
@@ -769,6 +771,14 @@ class SwiftASTContext : public TypeSystem {
 
   void SetCachedType(const ConstString &mangled, const lldb::TypeSP &type_sp);
 
+  static bool
```

Comment:
Is there any reason why these have to be static methods?  They have to have a swift_ast_context to work, and they are in SwiftASTContext.h.  Why not make them instance methods?

Reply:
You are correct, I'll constify `SwiftASTContext` here.

Reply:
Not sure I understand this argument.  This function is loading a module into the AST context.   Conceptually this is not a const operation even if the compiler can't tell that.